java/ql/lib/change-notes/released/0.3.3.md

```diff
@@ -4,3 +4,4 @@
 
 * Improved analysis of the Android class `AsyncTask` so that data can properly flow through its methods according to the life-cycle steps described here: https://developer.android.com/reference/android/os/AsyncTask#the-4-steps.
 * Added a data-flow model for the `setProperty` method of `java.util.Properties`. Additional results may be found where relevant data is stored in and then retrieved from a `Properties` instance.
+* Added data-flow models for the `java.lang.String.(charAt|getBytes)` methods. Improved data-flow models for the `java.lang.String.(replace|replaceFirst|replaceAll)` methods. Additional results may be found where users do not properly sanitize their inputs.
```

Comment:
This shouldn't go under `released`, but rather you'll need to create a new change-note file under `lib/change-notes`. See https://github.com/github/codeql/blob/main/docs/change-notes.md.

Reply:
Got it.

Reply:
I labeled it as `minorAnalysis`. Not sure whether that's the correct category.

Reply:
Sounds good, that's what I would've labeled as too.